sdc-plugin/tests/pll/pll.tcl

```diff
@@ -0,0 +1,23 @@
+yosys -import
+plugin -i sdc
+# Import the commands from the plugins to the tcl interpreter
+yosys -import
+
+read_verilog pll.v
+read_verilog -specify -lib -D_EXPLICIT_CARRY +/xilinx/cells_sim.v
+read_verilog -lib +/xilinx/cells_xtra.v
+hierarchy -check -auto-top
+
+# Start flow after library reading
+synth_xilinx -vpr -flatten -abc9 -nosrl -nodsp -iopad -run prepare:check
+
+# Read the design timing constraints
+read_sdc $::env(INPUT_SDC_FILE)
+
+# Propagate the clocks
+propagate_clocks
```

Comment:
Why does it need to be called 3 times?

Reply:
Explained it [here](https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1647#discussion_r476605300)